content/client-server-api/modules/instant_messaging.md

```diff
@@ -327,11 +327,12 @@ If a reason were to be supplied, it would look like:
 }
 ```
 
-When sending a spoiler, clients SHOULD provide the plain text fallback in the `body`
+When sending a spoiler, clients SHOULD provide the spoilered text fallback in the `body`
```

Comment:
The term "spoilered" is ambiguous for me. Does it mean the un-obfuscated text (ie, containing spoilers), or does it mean text which has been though an anti-spoilering process?

Reply:
The text containing spoilers, but good point. Replacing simply with "the text containing spoilers" is much clearer.